doc/gemini_gmos.rst

```diff
@@ -10,6 +10,14 @@ This file summarizes several instrument specific
 settings that are related to the Gemini/GMOS spectrograph.
 
 
+Nod and Shuffle
```

Comment:
Can we add a GitHub issue saying that we need to eventually do this right.

Reply:
our sky subtraction is so good, you will likely only increase
the error "doing it right"